### DISCUSSION TITLE
There seems to be a bug in free@home with forced positions

### DISCUSSION THREAD

**Discussion by nrenner73** [kingsleyadam/local-abbfreeathome-hass#112] [Q&A] [answered]
This has nothing to do with the HA integration at all. I just put it here because I don't know what to think of it. 
I was playing with the forced_close option of my attic window when I realized this. So I went to the free@home app and the behavior is the same. 

When the window is open in any position and I do a forced close it closes as it should and then both the button in the app and the wall switch are disabled. The wall switch also starts blinking to indicate the forced position. 
All as it should be. 

But when I deactivate the forced lock again, the window opens back to its former position. Regardless of the setting. The setting in the actuator would give me the option to either remain on position or move back to the previous position after a forced lock is lifted. 

Are you able to reproduce that behavior?

What can I do to make this known to ABB/Busch Jaeger? Does someone of you have any contact?

Or is it just me and I didn't get the trick?

Thanks

**Comment by kingsleyadam** [comment, marked as answer]
I don’t have any covers, but this is the same behavior with lights. When disabling the forced position it sets the light back to it’s previous state before forced being set. So I’d call this as designed by the ABB-free@home team.
